backend/src/main/java/gov/cdc/usds/simplereport/service/PatientBulkUploadServiceAsync.java

```diff
@@ -164,7 +167,7 @@ public CompletableFuture<Set<Person>> savePatients(byte[] content, UUID facility
             EmailProviderTemplate.SIMPLE_REPORT_PATIENT_UPLOAD_ERROR,
             Map.of("simplereport_url", simplereportUrl));
 
-        String errorMessage = "Error uploading patient roster";
+        String errorMessage = "Error uploading patient roster " + e;
```

Comment:
is there a possibility that the exception might contain PII or security details?

Reply:
this is an interesting point. if it did contain PII, is it an issue to be logging this? do we already log PII in our audit logs?